src/client/components/MainNavBar/LogoutLink.tsx

```diff
@@ -12,19 +13,25 @@ interface LogoutLinkProps {
 export default function LogoutLink({ children }: LogoutLinkProps) {
   const session = useSessionValue();
   return (
-    <Linkd
+    <a
       onClick={(event) => {
         event.preventDefault();
         session.logout();
         return false;
       }}
-      lean={true}
-      className={styles.LogoutLink}
-      icon={IconLogout}
+      className={classNames(
+        buttonStyle({
```

Comment:
`buttonStyle` is al een wrapper om de `classNames` functie het lijkt me dan niet nodig om de output daarvan weer in een `classNames` functie te stoppen.